src/ray/util/log_redirection_options.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2025 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <limits>
+#include <string>
+
+namespace ray {
+
+// Configuration for log redirection. By default no rotation enabled.
+//
+// It's allowed to have multiple sinks are enabled in the option.
+// For example, if redirection file set and `tee_to_stdout` both set to true, the stream
+// content is written to both sinks.
+struct LogRedirectionOption {
```

Comment:
StreamRedirectionOption is better since you are calling stdout, stderr streams elsewhere

Reply:
updated